README.md

```diff
@@ -34,10 +34,13 @@ Install `fixed-data-table-2` using npm.
 ```shell
 npm install fixed-data-table-2
 ```
-Add the default stylesheet `dist/fixed-data-table.css`, then import it into any module.
+Add the default stylesheet `dist/fixed-data-table.css` using a link tag or import it with a CSS module.
 
 ### Basic Example
-
+Note this example demonstrates typical FDT usage, but won't work out of the box.  
```

Comment:
I think we should only include the MyCustomCell that you have. Maybe show the code inline. The rest of this seems excessive